test/sanity/validate-modules/schema.py

```diff
@@ -83,23 +83,51 @@ def return_schema(data):
     )
 
 
+def deprecation_schema():
+
+    deprecation_schema_dict = {
+        # Every time we branch:
+        #   * old versions should be removed from the beginning of this list
+        #   * git rm old modules
+        #   * Update CHANGELOG & porting_guide
+        Required('version'): Any("2.0", "2.2", "2.3", "2.4", "2.5"),
+        Required('why'): Any(*string_types),
+        Required('alternative'): Any(*string_types),
+        'removed': Any(True),
```

Comment:
I don't think we need a `removed=True`.  With docs being versioned, there should never be a need to keep that info.

Reply:
Agreed, removed from here and the only module it's been used in